ModTek/Logging/BetterLogHandler.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using System.IO;
+using HBS.Logging;
+using Logger = HBS.Logging.Logger;
+using Object = UnityEngine.Object;
+
+namespace ModTek.Logging
+{
+    internal class BetterLogHandler
+    {
+        internal static BetterLogHandler Shared = new BetterLogHandler();
+
+        private ILogAppender cleanAppender;
+        private LogLevel logLevel;
+        //private bool ignoreLoggerLogLevel = true; // logger log level is hidden
```

Comment:
for the cleaned log, we unfortunately ignore the original loggers log level, as thats hidden behind a recursive working property. so the cleaned log really includes everything from any logger, regardless if the logger would even log it.